cmake/define.inc

```diff
@@ -150,6 +150,14 @@ IF (TD_BUILD_HTTP)
   ADD_DEFINITIONS(-DHTTP_EMBEDDED)
 ENDIF ()
 
+IF (${BUILD_LUA} MATCHES "true")
```

Comment:
as we talked about, we need to make LUA working by default for UDF functionality.